components/ntp_background_images/browser/view_counter_model_unittest.cc

```diff
@@ -19,22 +22,29 @@ namespace ntp_background_images {
 
 class ViewCounterModelTest : public testing::Test {
  public:
-  ViewCounterModelTest() = default;
+  ViewCounterModelTest()
+      : task_environment_(base::test::TaskEnvironment::TimeSource::MOCK_TIME) {}
   ~ViewCounterModelTest() override = default;
 
   void SetUp() override {
     auto* registry = prefs()->registry();
     ViewCounterService::RegisterProfilePrefs(registry);
+
+    base::FieldTrialParams parameters;
+    std::vector<base::test::FeatureRefAndParams> enabled_features;
+    parameters[features::kInitialCountToBrandedWallpaper.name] = "1";
+    parameters[features::kCountToBrandedWallpaper.name] = "4";
```

Comment:
I think we need to use same value with production? 
```suggestion
    parameters[features::kInitialCountToBrandedWallpaper.name] = "0";
    parameters[features::kCountToBrandedWallpaper.name] = "3";
```

Reply:
fixed, but only `kCountToBrandedWallpaper` since `kInitialCountToBrandedWallpaper` is no longer being changed